frontend/src/services/analytics.js

```diff
@@ -4,18 +4,30 @@ export class Analytics {
     constructor(){}
 
     ga(){
-        const url = '/api/analytics/ga'
-        return axios.get(url, {withCredentials: true}).then(response => response.data)
+        const url = '/api/an/ga'
+        try{
+            return axios.get(url, {withCredentials: true}).then(response => response.data)
+        }catch(ex){
+            return ''
```

Comment:
Do these need to return an actual error?

Reply:
No it only errors when it's been blocked by an analytics blocker this stops it from crashing, and no one really cares if the tracking failed.